django/library/views.py

```diff
@@ -12,29 +13,26 @@
 
 class CodebaseViewSet(viewsets.ModelViewSet):
     lookup_field = 'identifier'
-    multiple_lookup_fields = ('identifier', 'uuid', 'pk')
+    lookup_url_kwarg = 'pk'
```

Comment:
we should use the default `identifier` here to not break the expectation that the lookup kwarg matches the underlying model attribute